src/hotspot/share/prims/jvmtiThreadState.cpp

```diff
@@ -379,10 +461,6 @@ JvmtiVTMSTransitionDisabler::start_VTMS_transition(jthread vthread, bool is_moun
   // Enter VTMS transition section.
   assert(!thread->is_in_VTMS_transition(), "VTMS_transition sanity check");
   thread->set_is_in_VTMS_transition(true);
-  JvmtiThreadState* vstate = java_lang_Thread::jvmti_thread_state(vth());
-  if (vstate != NULL) {
-    vstate->set_is_in_VTMS_transition(true);
```

Comment:
Is the VTMS transition flag in the `JvmtiThreadState` dead code now?

Reply:
Yes. All this dead code has been removed now.